labellab-client/src/components/project/images.js

```diff
@@ -2,7 +2,14 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
-import { Table, Button, Form, Dimmer, Loader, Icon } from 'semantic-ui-react'
+import {
+  Table,
+  Button,
+  Form,
+  Dimmer,
+  Loader,
+  Checkbox
```

Comment:
Nice work 👍🏻, but you missed an ```Icon``` import from semantic which results in an error on line 264